app/models/spotlight/search.rb

```diff
@@ -52,22 +50,30 @@ def images
   end
 
   def as_json(*args)
-    super.merge(featured_image: featured_image, count: count)
-  end
-
-  def default_featured_item_id
-    images.first.first if images.present?
+    super.merge(count: count)
```

Comment:
Can we ditch this override/super in favor of adding #count to the `methods` options passed to #as_json in https://github.com/sul-dlss/spotlight/pull/997/files#diff-dd6db6608a4fa90bcf92593e2296f214R23

I can't remember if that's the only place this is used or not.